core/core.go

```diff
@@ -200,12 +200,24 @@ func GetEnvOrConfig(name string) string {
 	return fileConfig[name]
 }
 
+// isValidWorkspace returns true iff the supplied path is the workspace root, defined by the presence of
+// a file named WORKSPACE or WORKSPACE.bazel
+// see https://github.com/bazelbuild/bazel/blob/8346ea4cfdd9fbd170d51a528fee26f912dad2d5/src/main/cpp/workspace_layout.cc#L37
+func isWorkspaceRoot(path string) bool {
```

Comment:
Either this function has to be called `isValidWorkspace` or the function calls below have to be changed to `isWorkspaceRoot`.

Reply:
Yeah, just fixed it. I edited through github so I didn't see that I forgot to rename the function after changing the call sites.